src/extensions/common-api/utils.ts

```diff
@@ -13,7 +13,7 @@ import type { IClassName } from "../../renderer/utils/cssNames";
 import { cssNames } from "../../renderer/utils/cssNames";
 
 export interface UtilsExtensionItems {
-  Singleton: Singleton;
+  Singleton: typeof Singleton;
```

Comment:
The rest here are functions so they are ok(?)

Reply:
I believe so.